src/test/datascience/mockInputBox.ts

```diff
@@ -38,27 +41,31 @@ export class MockInputBox implements InputBox {
         return this.didHideEmitter.event;
     }
     public show(): void {
-        // After 10 ms set the value, then accept it
-        setTimeout(() => {
-            this.value = this._value;
-            this.didChangeValueEmitter.fire(this._value);
+        // Unless we have specified more, we only want to fire the input one time (in case show is called more)
```

Comment:
Tweak to get unit tests working. Due to the validation change above show can be called more often. Show has been rigged in the mock to input and fire the value as if a user entered it. Instead, just do it once (with the option for more for one test that expected to call it multiple times with user input).